CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+# Contributing
+
+Shamelessly taken from the [coc.nvim](https://github.com/neoclide/coc.nvim) project with sincere thanks.
+
+## How do I... <a name="toc"></a>
+
+- [Use This Guide](#introduction)?
+- Make Something? 🤓👩🏽‍💻📜🍳
+  - [Project Setup](#project-setup)
+  - [Contribute Documentation](#contribute-documentation)
+  - [Contribute Code](#contribute-code)
+- Manage Something ✅🙆🏼💃👔
+  - [Provide Support on Issues](#provide-support-on-issues)
+  - [Review Pull Requests](#review-pull-requests)
+  - [Join the Project Team](#join-the-project-team)
+
+## Introduction
+
+Thank you so much for your interest in contributing!. All types of contributions are encouraged and valued. See the [table of contents](#toc) for different ways to help and details about how this project handles them!📝
+
+The [Project Team](#join-the-project-team) looks forward to your contributions. 🙌🏾✨
+
+## Project Setup
+
+So you wanna contribute some code! That's great! This project uses GitHub Pull Requests to manage contributions, so [read up on how to fork a GitHub project and file a PR](https://guides.github.com/activities/forking) if you've never done it before.
+
+If this seems like a lot or you aren't able to do all this setup, you might also be able to [edit the files directly](https://help.github.com/articles/editing-files-in-another-user-s-repository/) without having to do any of this setup. Yes, [even code](#contribute-code).
+
+### Run Project Locally
+
+If you want to go the usual route and run the project locally, though:
+
+- [Install PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell)
+- [Install Node.js](https://nodejs.org/en/download/)
+- [Install npm](https://npmjs.com)
+- [Fork the project](https://guides.github.com/activities/forking/#fork)
+
+Then in your (PowerShell) terminal:
+
+- Add coc-powershell to your vim's rtp by `set runtimepath^=/path/to/coc-powershell`
+- `cd path/to/your/coc-powershell`
+- `./build.ps1`
+
+### Run Project Within a Docker Container
```

Comment:
Let's leave this out for now since the Dockerfile isn't quite ready yet.

Reply:
Oh I suppose, you know that was like 30 seconds of typing you're asking me to undo... 😂